spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe API::V1::KeywordsController, type: :request do
+  describe '#index' do
+    context 'given an unauthenticated user' do
+      it 'returns a 401 status code' do
+        get :index
+
+        expect(response.status).to eq(401)
+      end
+    end
+
+    context 'when no keywords' do
```

Comment:
can we change it to 'when there are no keywords', or 'when keyword_list is empty'?

Reply:
Sure! :) Done ;-)